docs/user-guides/cluster/oidc_connectivity.md

```diff
@@ -0,0 +1,179 @@
+in structured authentication configuration.
+
+check out https://kubernetes.io/docs/reference/access-authn-authz/authentication/#using-authentication-configuration
+for more information 
+```
+
+Configure the OIDC issuer in the `Structured Authentication Configuration` of the remote cluster
+
+**Example Structured Authentication Configuration file**
+
+```yaml
+apiVersion: apiserver.config.k8s.io/v1beta1
+kind: AuthenticationConfiguration
+jwt:
+  - issuer:
+      url: https://<greenhouse-service-account-issuer>
+      audiences:
+        - greenhouse # audience should be greenhouse
+    claimMappings:
+      username:
+        claim: 'sub' # claim to be used as username
+        prefix: 'greenhouse:' # prefix to be added to the username to prevent impersonation (can be any string of your choice)
+# additional trusted issuers
+#     - issuer:
+```
+
+Add RBAC rules to the remote cluster to allow the `Greenhouse` cluster to be authorized to CRUD kubernetes resources
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: greenhouse-<cluster-name>-oidc-access
+subjects:
+  - kind: User
+    apiGroup: rbac.authorization.k8s.io
+    name: greenhouse:system:serviceaccount:<your-organization-namespace>:<cluster-name>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+```
+
+The subject kind `User` name must follow the pattern of
+`<prefix>:system:serviceaccount:<your-organization-namespace>:<cluster-name>`
+
+`<prefix>` is the prefix used in the `Structured Authentication Configuration` file for the `username` claim mapping
+
+```NOTE
+For convenience purposes, the `prefix` is set to `greenhouse:` in the example `Structured Authentication Configuration`
+but it can be any string identifier of your choice
+
+if you use '-' in prefix, for example, `identifier-` then the subject name should be `identifier-system:serviceaccount:<your-organization-namespace>:<cluster-name>`
+```
+
+### Onboard
+
+You can now onboard the remote cluster to your Greenhouse organization by applying a secret in the following format:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    "oidc.greenhouse.sap/api-server-url": "https://<remote-cluster-api-server-url>"
+  name: <cluster-name> # ensure the name provided here is the same as the <cluster-name> in the ClusterRoleBinding
+  namespace: <organization-namespace>
+data:
+  ca.crt: <double-encoded-ca.crt> # remote cluster CA certificate base64 encoded
+type: greenhouse.sap/oidc # secret type
+```
+
+Mandatory fields:
+
+- the annotation `oidc.greenhouse.sap/api-server-url` must have a valid URL pointing to the remote cluster's API server
+- the `ca.crt` field must contain the remote cluster's CA certificate
+- the secret type must be `greenhouse.sap/oidc`
+- the secret name must be the same as the `<cluster-name>` used in the `ClusterRoleBinding`
```

Comment:
```suggestion
- the name fo the Secret must equal the `<cluster-name>` used in the `ClusterRoleBinding`
```

Reply:
It should be mentioned early on in the documentation, that the `<cluster-name>` will be used for the Greenhouse resource and that it is imutable